data.sql

```diff
@@ -46,4 +46,41 @@ WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
 
 UPDATE animals
 SET owner_id = (SELECT id FROM owners WHERE fullname = 'Dean Winchester')
-WHERE name IN ('Angemon', 'Boarmon');
\ No newline at end of file
+WHERE name IN ('Angemon', 'Boarmon');
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES
+('Williams Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations (vets_id, species_id)
+VALUES
+(1, 1),
+(2, 2),
```

Comment:
- [ ] Let us kindly address this part of the data insertion. Here is what we are expected to input as seen in the screenshot below
![Screenshot from 2023-05-08 08-38-49](https://user-images.githubusercontent.com/46388113/236776182-0206c2f6-973c-499c-908c-0b59a8c1494b.png)
According to the data, hardcoded (2,2,) means vets_id(2) = Maisy Smith instead of Stephanie Mendez